p2p/protocol/circuitv2/pb/voucher.proto

```diff
@@ -1,9 +1,9 @@
-syntax = "proto2";
+syntax = "proto3";
 
 package circuit.pb;
 
 message ReservationVoucher {
-  required bytes relay = 1;
-  required bytes peer = 2;
-  required uint64 expiration = 3;
+  bytes relay = 1;
```

Comment:
These need to be optional, no? That's what the specs say

Reply:
Sorry, I missed this. Updated now to reflect https://github.com/libp2p/specs/pull/511/files?short_path=5d2c9cb#diff-5d2c9cb6bfb3274dc65770bf6735c8d6c343245aaf1d9fac474df154842b81d2